src/main/java/com/faforever/moderatorclient/ui/MessagesController.java

```diff
@@ -0,0 +1,102 @@
+package com.faforever.moderatorclient.ui;
+
+import com.faforever.moderatorclient.api.domain.MessagesService;
+import com.faforever.moderatorclient.ui.domain.MessageFx;
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import javafx.collections.transformation.SortedList;
+import javafx.fxml.FXML;
+import javafx.scene.Scene;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.HBox;
+import javafx.stage.Stage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.function.Predicate;
+
+@Component
+@Slf4j
+public class MessagesController {
+    public HBox root;
+    public ToggleGroup filterGroup;
+    public TableView<MessageFx> messageTableView;
+    private final MessagesService messagesService;
+    private final UiService uiService;
+    public TextField filter;
+    public RadioButton valueRadioButton;
+    public RadioButton keyRadioButton;
+    public RadioButton regionRadioButton;
+    public RadioButton languageRadioButton;
+    public RadioButton noneRadioButton;
+    private FilteredList<MessageFx> filteredItemList;
+    private ObservableList<MessageFx> messageFxes;
+
+    @Inject
+    public MessagesController(MessagesService messagesService, UiService uiService) {
+        this.messagesService = messagesService;
+        this.uiService = uiService;
+    }
+
+    @FXML
+    public void initialize() {
+        ViewHelper.buildTutorialTable(messageTableView, messagesService, log);
+        onRefreshMessages();
```

Comment:
Don't invoke loading of data in @FXML initialize.  Call them manually from MainController. This is due to error handling. We want to throw errors first after the user passed the login window (this is new behavior in develop).